library/eclipse-mosquitto

```diff
@@ -1,7 +1,11 @@
 Maintainers: David Audet <david.audet@ca.com> (@daudetCA)
 GitRepo: https://github.com/eclipse/mosquitto.git
 
-Tags: 1.4.10, latest
+Tags: 1.4.12, latest
+GitCommit: 0bb602ed7a89da80d25e6b959a130fdcf0556be5
+Directory: docker/1.4.12
+
+Tags: 1.4.10
```

Comment:
If it's important for users to know it's still available and supported, it's fine to leave it here, but if it's not going to receive any further updates, it's likely better to remove it from the supported list to make that signal clear, right?